tasks/sep-dev-0/005-holocene-proofs/.env

```diff
@@ -0,0 +1,4 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+SUPERCHAIN_CONFIG_ADDR=0xC2Be75506d5724086DEB7245bd260Cc9753911Be
```

Comment:
This doesn't match the value in the superchain registry: https://github.com/ethereum-optimism/superchain-registry/blob/c8ef19dc6663aa337b72d2191abbc03681379b3b/superchain/configs/sepolia-dev-0/superchain.toml#L3
This looks like the sepolia testnet address.